src/Adapter/Product/QueryHandler/GetPackedProductsHandler.php

```diff
@@ -36,7 +36,7 @@
 /**
  * Handles GetPackedProducts query using legacy object model
  */
-class GetPackedProductsHandler implements GetPackedProductsHandlerInterface
+final class GetPackedProductsHandler implements GetPackedProductsHandlerInterface
```

Comment:
I wonder what is the criteria for us to decide if a handler is final or not? 🤷‍♂️

Reply:
i remember only one clear criteria - if a class implements interface then it can be final.

Reply:
I think our Handlers are built with decoration in mind, not extension 👍  so it's good

Reply:
@PierreRambaud close to the "private VS protected" properties debate, we might have a "final or not" debate about our classes 🤔  wdyt ?